src/Terminal/HelpText.ts

```diff
@@ -295,28 +295,30 @@ export const HelpTexts: IMap<string[]> = {
     " ",
   ],
   ls: [
-    "Usage: ls [dir] [| grep pattern]",
+    "Usage: ls [dir] [--grep pattern] | [-l]",
```

Comment:
The pipe would suggest only one of two is allowed.
Also to keep in line with other places, swap the two around.
Suggestion: `"Usage: ls [dir] [-l] [--grep pattern]"`